src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -450,6 +450,8 @@ public async Task<HttpResponseMessage> SendAsyncCore(HttpRequestMessage request,
                 }
                 else
                 {
+                    if (NetEventSource.IsEnabled) Trace("Request content is not null. Start sending it.");
```

Comment:
When expect 100-continue is enabled, we will not be sending the content at this point, and may end up not sending it at all. Just to avoid any potentially misleading logs, I think we should move this.

Reply:
This log should apply for expect 100-continue case as well: we start the process of preparing it, doesn't necessarily mean we should send it now. This can be helpful when we want to diagnose if the SendAsync task hang/get cancelled, and we are at which stage.

I will make the log clearer.